modules/ROOT/partials/configuration/link_default_target.adoc

```diff
@@ -3,17 +3,23 @@
 
 This option allows you to set a default `+target+` value for links when inserting/editing a link via the link dialog. If the value of `+link_default_target+` matches a value specified by the xref:link.adoc#link_target_list[`+link_target_list+`] option, that item will be set as the default item for the targets dropdown in the link dialog.
 
+// TODO: Should this be shown if already on the autolink page?
 NOTE: This option also applies to the xref:autolink.adoc[autolink] plugin.
 
 Type: `+String+`
 
 === Example: Using `+link_default_target+`
 
-[source,js]
+:plugin_list: link
+ifeval::["{plugincode}" == "autolink"]
+:plugin_list: link autolink
+endif::[]
```

Comment:
- [ ]  Move to the top of the file (see previous comment)

Reply:
Done